rt/bash/runsort

```diff
@@ -114,6 +111,9 @@ fi
 
 # done with tests
 
+# remove residual files from SORT executions
+rm -f sort/SORT.ERR sort/SORT.LOG sort/SORT.STA
```

Comment:
You added a line in RUNSORT.BAT that deletes SORTWK*.TMP. Please add it here. Here's the line to add:
```
rm -f SORTWK*.TMP
```